common/scripts/Ubuntu_14.04/installWorker.sh

```diff
@@ -57,7 +57,11 @@ __initialize_worker() {
 __registry_login() {
   echo "Logging into registry"
   mkdir -p ~/.aws
-  local docker_login_cmd=$(aws ecr --region us-east-1 get-login)
+  if [ "$NO_VERIFY_SSL" == true ]; then
```

Comment:
This will gets called by `common/scripts/installWorker.sh` below. I'm not sure why login is required twice. I have kept the existing behavior as-is.